### ISSUE TITLE
Add a license

### ISSUE THREAD

**Issue by bellerophons-pegasus** [dhtaxonomy/TaDiRAH#24]
I think a license should be included for the taxonomy. I would suggest to use CC-BY (https://f.hypotheses.org/wp-content/blogs.dir/5978/files/2019/02/Letak-ELDAH_24_11_2.pdf) or be bold and go for Public Domain.

**Comment by arockenberger** [comment]
I second that!

**Comment by christofs** [comment]
True, thanks for pointing out the oversight. We'll talk about it in the team, but CC BY sounds good, even more open would be fine with me as well.

**Comment by arockenberger** [comment]
That would be awesome! I'm using the taxonomy here at Uni Oslo Library (and I already made some visualizations) and I would like to officially put them somewhere. We are also working on a translation (into Norwegian).
